### PR TITLE
docs: BitBucket Runner Trigger Plugin Doc

### LINKED ISSUE #19: DOC: Create a documentation for BitBucket Runner Trigger plugin
Documentation for the BitBucket Runner Trigger plugin is not present in docs, it needs to be created and added in docs.

### PR DESCRIPTION
**Pull request by bhushan-nemade-dt**
Fix #19

### CODE REVIEW

**Review by abhibhaw — DISMISSED** [review]
LGTM

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -132,6 +132,7 @@
 * [Pipeline Plugins](user-guide/plugins/README.md)
   * [Create Your Plugin](user-guide/plugins/create-plugin.md)
   * [Our Plugins](user-guide/plugins/plugin-list.md)
+    * [BitBucket Runner Trigger](user-guide/plugins/bitbucket-runner-trigger.md)
```
Comment (ashoknayak777) [inline docs/SUMMARY.md]: Replace `BitBucket` with `Bitbucket`
We need to get the capitalization fixed by the devs too.

File: docs/user-guide/plugins/bitbucket-runner-trigger.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# BitBucket Runner Trigger
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/bitbucket-runner-trigger.md]: Except the variables, replace all instances of `BitBucket` with `Bitbucket` in our doc.
Again, we need to request the devs to fix it in the plugin name and its subtext.

File: docs/user-guide/plugins/bitbucket-runner-trigger.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Introduction
+The **BitBucket Runner Trigger** plugin by Devtron enables integration between Devtron CI/CD workflows and BitBucket Runners. It allows users to remotely trigger and manage BitBucket Runners directly from within their Devtron pipelines, enhancing automation capabilities and workflow control.
+
+### Prerequisites
+Before integrating the **BitBucket Runner Trigger** plugin, ensure you have an BitBucket account, properly configured BitBucket Runner, and BitBucketToken with appropriate permissions.
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/bitbucket-runner-trigger.md]: `BitBucketToken` should be `Bitbucket token`

Comment (ashoknayak777) [reply docs/user-guide/plugins/bitbucket-runner-trigger.md]: Replace 'an' with 'a' in: `you have an BitBucket account`

File: docs/user-guide/plugins/bitbucket-runner-trigger.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Add a brief explanation of the task and the reason for choosing the plugin. Include information for someone else to understand the purpose of the task.
+
+e.g., `The BitBucket Runner Trigger plugin is integrated to trigger an BitBucker Runner remotely through Devtron CI/CD workflow.`
+
+### Input Variables
+
+| Variable                 | Format       | Description | Sample Value |
+| ------------------------ | ------------ | ----------- | ------------ |
+|   BitBucketWorkspaceName | STRING       | The workspace name in BitBucket where your repository is located | dev-workspace             |
+|  BitBucketUsername       | STRING       | Your BitBucket username used for authentication (Mandetory when pipeline is configured through SSH) | john.doe    |
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/bitbucket-runner-trigger.md]: Typo: `Mandatory`

File: docs/user-guide/plugins/bitbucket-runner-trigger.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+|   BitBucketToken         | STRING       | BitBucket access token for API authentication (Mandetory when pipeline is configured through SSH)  | eyJ0eXBlIjoic2VydmljZV9hY2             |
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/bitbucket-runner-trigger.md]: Typo: `Mandatory`

File: docs/user-guide/plugins/bitbucket-runner-trigger.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Task Name
+Enter the name of your task
+
+e.g., `BitBucket Runner Trigger`
+
+### Description
+Add a brief explanation of the task and the reason for choosing the plugin. Include information for someone else to understand the purpose of the task.
+
+e.g., `The BitBucket Runner Trigger plugin is integrated to trigger an BitBucker Runner remotely through Devtron CI/CD workflow.`
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/bitbucket-runner-trigger.md]: Replace 'an' with 'a' in: `trigger an BitBucker Runner`

**Review by ashoknayak777 — CHANGES_REQUESTED** [review]
@bhushan-nemade-dt, let's incorporate the suggested fixes

File: docs/SUMMARY.md (hunk 1 above)
Comment (bhushan-nemade-dt) [reply docs/SUMMARY.md]: fixed

File: docs/user-guide/plugins/bitbucket-runner-trigger.md (hunk 2 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/bitbucket-runner-trigger.md]: changed `BitBucket` to `Bitbucket` in doc

File: docs/user-guide/plugins/bitbucket-runner-trigger.md (hunk 3 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/bitbucket-runner-trigger.md]: done

**Review by ashoknayak777 — APPROVED** [review]
LGTM

**Review by abhibhaw — APPROVED** [review]
LGTM

**Pull request merged**